### DISCUSSION TITLE
sensor cleanup function

### DISCUSSION THREAD

**Discussion by garadar** [StackStorm/st2#5798] [Packs]
## context
```
(cluster)-[xxx@auto ~]$ cat /etc/centos-release
CentOS Linux release 7.9.2009 (Core)
(cluster)-[xxx@auto ~]$ uname -r 
3.10.0-1160.76.1.el7.x86_64
(cluster)-[xxx@auto ~]$  st2 --version
st2 3.7.0, on Python 3.6.8
```

## Issue
_Sensor_
I'm using a flask server to listen my events and when I turn off st2 or disable the sensor, the function cleanup is never called.

The cleanup function is called when there is an issue with stackstorm main entity.

Do you have an idea ?

**Comment by arm4b** [comment]
`stackstorm/community` is about Plans, Proposals, RFCs and Community

Moved this question to the stackstorm/st2 repository under Discussions.

**Comment by garadar** [comment]
Hello there is someone can help me for this ?
